app/views/devise/shared/_flashes.html.erb

```diff
@@ -5,7 +5,10 @@
 <% end %>
 
 <% if alert %>
-  <div class="alert">
-    <%= alert %>
+  <div class="alert-container" data-controller="hide-devise-alert" data-hide-devise-alert-target="alert">
+    <div class="alert">
+      <p>Ops, something went wrong <i class="fa-regular fa-face-sad-tear"></i></p>
```

Comment:
Please, replace Ops with Oops or Ups

Reply:
Ok, leave it to me

Reply:
Actually I'm not even sure you should be adding any message as you already render `<%= alert %>`